packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -66,8 +66,10 @@ export async function prefetchLatestSnapshot(
 	snapshotFormatFetchType?: SnapshotFormatSupportType,
 	odspDocumentServiceFactory?: OdspDocumentServiceFactory,
 ): Promise<boolean> {
-	const odspLogger = createOdspLogger(
-		createChildLogger({ logger, namespace: "PrefetchSnapshot" }),
+	const mc = createChildMonitoringContext({ logger, namespace: "PrefetchSnapshot" });
+	const odspLogger = createOdspLogger(mc.logger);
+	const useGroupIdsForSnapshotFetch = mc.config.getBoolean(
+		"Fluid.Container.FetchSnapshotUsingGetSnapshotApi",
```

Comment:
To be honest, looking at FG name, I do not know what Fluid.Container.FetchSnapshotUsingGetSnapshotApi means and what should happen.
I also not sure why we need any control here. IMHO, prefetch should be as normal load. If we are talking about 1GB files with default snapshot of 1MB file, then fetching full file will make things so much slower. Maybe there are scenarios like "fetch snapshot 5 min before meeting" where fetching whole thing is better, but I think we will run into a lot of problems. I'd rather say - it's same thing, and if app wants something different, they better just load the whole container instance and it will do right things in terms of making decisions to load (in idle) more groups, and possibly we will build (in the future) some caching for them.
We should assume that container will get rid of groups that it does not need (from memory), so fetching more is not necessarily better, at least when we talk about infinite content.

Reply:
Fixed the FG name. Yes, I am doing same, where prefetch just fetched the ungrouped data.